pkgs/development/python-modules/insteon-frontend-home-assistant/default.nix

```diff
@@ -17,8 +19,18 @@ buildPythonPackage rec {
     hash = "sha256-R+P4pgKbLvf0mwpSDoujCvlJe/yS+nvSJ7ewLVOOg/0=";
   };
 
+  patches = [
+    # https://github.com/pyinsteon/insteon-panel/pull/33
+    (fetchpatch {
+      name = "unpin-setuptools.patch";
+      url = "https://github.com/pyinsteon/insteon-panel/commit/2297eb05668907edd03633f244e5876990e340c7.patch";
+      hash = "sha256-kTu1+IwDrcdqelyK/vfhxw8MQBis5I1jag7YTytKQhs=";
+    })
+  ];
+
   nativeBuildInputs = [
     setuptools
+    wheel
```

Comment:
Why add wheel given that the patch removes it?

Reply:
Good question. The patch removes declaring it as a build dependency in pyproject.toml but it doesn't mean that wheel isn't a dependency. [PEP 517](https://peps.python.org/pep-0517/) describes that a build backend like setuptools can declare additional build dependencies via the `get_requires_for_build_wheel` and `get_requires_for_build_sdist` methods, which setuptools does. It's best practice not to include wheel as a dependency in pyproject.toml because then it will be downloaded even in an sdist build (though that doesn't apply to us in nixpkgs).